jakartaee-microprofile-example/README.md

```diff
@@ -50,33 +50,33 @@ Use the Maven wrapper to start the application by using the [Liberty dev mode](h
 
 ### Try out other models
 
-Visit the http://localhost:9080/openapi/ui URL (OpenAPI UI) for the following 3 REST APIs:
+Navigate to the the [OpenAPI UI](http://localhost:9080/openapi/ui) URL for the following 3 REST APIs:
 
 - [HuggingFaceLanguageMode](https://github.com/langchain4j/langchain4j/blob/main/langchain4j-hugging-face/src/main/java/dev/langchain4j/model/huggingface/HuggingFaceLanguageModel.java)
-  - expand the GET `/api/model/language` API
-    - click the `Try it out` button
-    - type `When was langchain4j launched?` or any question on the question field
-    - click the `Execute` button
-  - or run the following `curl` command on a command-line session:
+  - Expand the GET `/api/model/language` API.
+    1. Click the **Try it out** button.
+    2. Type `When was langchain4j launched?`, or any question, in the question field.
+    3. Click the **Execute** button.
+  - Alternatively, run the following `curl` command from a command-line session:
     - ```
       curl 'http://localhost:9080/api/model/language?question=When%20was%20langchain4j%20launched%3F'
       ```
 - [HuggingFaceChatMode](https://github.com/langchain4j/langchain4j/blob/main/langchain4j-hugging-face/src/main/java/dev/langchain4j/model/huggingface/HuggingFaceChatModel.java)
   - expand the GET `/api/model/language` API
-    - click the `Try it out` button
-    - type `Which are the most popular?` or any question on the question field
-    - click the `Execute` button
-  - or run the following `curl` command on a command-line session:
+    1. Click the **Try it out** button.
+    2. Type `Which are the most popular?`, or any question, in the question field. 
```

Comment:
> Which are the most popular?

This doesn't seem like a good example question for an LLM- there's not context, ie which _what_ are most popular?

Reply:
Because there is a pre-set message, the question will make sense.
- see this [code block](https://github.com/gkwan-ibm/langchain4j-examples/blob/jee-mp-example/jakartaee-microprofile-example/src/main/java/dev/langchain4j/example/rest/ModelResource.java#L73-L100)
- the pre-set message at [line 90](https://github.com/gkwan-ibm/langchain4j-examples/blob/jee-mp-example/jakartaee-microprofile-example/src/main/java/dev/langchain4j/example/rest/ModelResource.java#L89-L90)